src/championOverview/ChampionOverview.tsx

```diff
@@ -58,23 +58,23 @@ const columns: ColumnDef<ChampionOverviewChampion, any>[] = [
     columnHelper.accessor((row) => row.totalGames, {
         id: 'totalGames',
         cell: (info) => info.getValue(),
-        header: () => <span>Total Games</span>,
+        header: () => <span>Total #</span>,

```

Comment:
@nsthi  recommended this change as a way to reduce the width of the table. 

i'm ok leaving this as games for now, but it will be increasingly a problem if we keep adding columns